scipy/spatial/distance.py

```diff
@@ -413,6 +422,34 @@ def sqeuclidean(u, v):
     return np.dot(u_v, u_v)
 
 
+def aitchison(u, v):
+    """
+    Computes the Aitchison distance between two 1-D arrays.
+    The Aitchison distance between 1-D arrays `u` and `v`, is defined as
+    .. math::
+       {||log \left( \frac{u}{v} \right) - mean \left(log \left( \frac{u}{v} \right) \right) ||}_2
```

Comment:
I have a hard time finding an authoritative definition. This paper has one: https://pdfs.semanticscholar.org/b0a9/530dfbebdbf774cdf24c442bf619236ee0f8.pdf - not specialized for 1-D, but also not immediately obvious that the definitions are the same. Do you have a reference?